libeppic/eppic_var.c

```diff
@@ -1249,10 +1249,8 @@ srcpos_t pos;
         eppic_rerror(&pos, "undefined variable '%s'", vn->name);
 
     }
-    if(!curv->ini && !insizeof && !vlev) {
-
+    if(!curv->ini && !curv->v && !insizeof && !vlev) {
```

Comment:
Why is this needed? Is a variable in fact initialized if curv->ini is zero and cur->v is non-null? How does it happen?